### DISCUSSION TITLE
is there any hack that gives all pets that is script based if so pls send me the link to the folder

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#977] [(D) Developer]
thanks

**Comment by hugo875** [comment]
`


const pets = gameData.pet.map(x =&gt; toPets(x.ID));	_.player.kennel.data.splice(-1, 0, ...pets);

`

**Reply by hugo875** [reply]
your welcome

**Reply by ghost** [reply]
eeer what script would i put in console for my pet team to be magmischief and big hex thanks
